graphprot/ResidueGraph.py

```diff
@@ -108,15 +108,18 @@ def get_graph(self, db):
 
         # create the interface edges
         for key, val in self.res_contact_pairs.items():
-            for v in val:
-                d = self._get_edge_distance(key, v, db)
-                self.nx.add_edge(key, v, dist=d, type=bytes(
-                    'interface', encoding='utf-8'))
+            if key in all_nodes :
```

Comment:
good catch ! You could make it even better by adding an else that throw an error or print a warning